docs/codeql/ql-language-reference/name-resolution.rst

```diff
@@ -216,15 +216,21 @@ Local namespaces
 In addition to the global module, type, and predicate namespaces, each module defines a number of local 
 module, type, and predicate namespaces.
 
-For a module ``M``, it's useful to distinguish between its **declared**, **exported**, and **imported** namespaces. 
-(These are described generically, but remember that there is always one for each of modules, types, and predicates.)
+For a module ``M``, it is useful to distinguish between its **privately declared**, **publically declared**, **exported**, and **visible** namespaces.
+(These are described generically, but remember that there is always one for each of modules, module signatures, types, type signatures, predicates, and predicate signatures.)
+
+- The **privately declared** namespaces of ``M`` contain all entities that are declared—that is, defined—in ``M`` and that are annotated as ``private``.
```

Comment:
Should we explcitly call out aliases now we talk about entities rather than names being declared.

Reply:
Sure! Talking about names no longer worked with shadowing (the names don't get hidden - the things with the name get hidden).
I've changed this to `entities and aliases` in the third commit. I suppose it could also be `declarations and aliases` but then it feels a bit self-referential.
Feel free to suggest a different formulation!

Reply:
Another option would also be all `entities that are declared or aliases in M`, in case you like that better?